packages/SidePanel/src/components/Header/Header.styles.js

```diff
@@ -0,0 +1,40 @@
+import { fontSize } from "@paprika/stylers/lib/helpers";
+import tokens from "@paprika/tokens/lib/tokens";
+import { css } from "styled-components";
+
+const kind = {
+  default: `background: ${tokens.color.white}; color: ${tokens.color.black};`,
+  primary: `background: ${tokens.color.purple}; color: ${tokens.color.white};`,
+};
+
+const space = Number.parseInt(tokens.space, 10);
+export const headerCSS = css`
+  border-bottom: 1px solid ${tokens.border.color};
+  box-sizing: border-box;
+  color: ;
+  display: flex;
+  justify-content: space-between;
+  padding: 12px ${space}px; // creates a 48px height
+  width: 100%;
+  align-items: center;
+
+  &:focus {
+    outline: 0;
+  }
+
+  .heading--level-1,
+  .heading--level-2,
+  .heading--level-3,
+  .heading--level-4,
+  .heading--level-5 {
+    ${fontSize()};
+    font-weight: 400;
```

Comment:
This concerns me. If we want to enforce the styling of the heading, maybe we can do it by just taking the content as a `string` and putting the `<Heading displayLevel={5} isLight>` component in ourselves.

I'm also starting to regret that we haven't implemented some kind of selector on the root node for each type of component... maybe not a `className`, but something would be convenient.

Reply:
That was my suggestion for `[data-component-type="listbox.header"]`

To me, that would super helpful you could target each component via jss, css or scss. as well in testing. even theme the components would be easier

Reply:
Yeah.  Let's make an issue to add these to all our components maybe? Then if there's a bigger conversation, it'll be separate from your PR.

Reply:
https://github.com/acl-services/paprika/issues/101 minor for this PR for now.

Reply:
btw about your previous comment? can we assume `displayLevel={5}` on the sidepanel? 🤔